src/data/primitives.json

```diff
@@ -1,4 +1,34 @@
 {
+  "borders": {
+    "radius": {
+      "base": {
+        "value": "8px",
+        "type": "borderRadius",
+        "description": "Default border radius option for most components including card-type components like NewBox, Accordion, etc."
+      },
```

Comment:
Removed the card version and updated base to be 8px from 4px.

Reply:
should we just change this to say `Box` ? under the assumption we are going to remove the `New` prefix? just one less thing to update later 🍅 🍅

Reply:
Yeah great call out! Done :)